leapp/libraries/stdlib/__init__.py

```diff
@@ -119,7 +116,37 @@ def _logging_handler(fd_info, a_buffer):
         __write_raw(fd_info, a_buffer)
 
 
-def run(args, split=False, callback_raw=_logging_handler, env=None, checked=True):
+def _logfile_logging_handler(fd_info, line):  # pylint: disable=unused-argument
+    """
+    Log into logfile
+
+    :param fd_info: Contains File descriptor type (not used)
+    :type fd_info: tuple
+    :param line: data to be printed to a logfile
+    :type line: string
+    """
+    def _raise_level(handlers):
+        for handler in handlers:
+            handler.setLevel(logging.DEBUG + 1)
```

Comment:
I would prefer not to have hard-coded DEBUG here but instead use level of handler you are setting

```suggestion
            handler.setLevel(handler.level + 1)
```